docs/gadgets/top/tcp.md

```diff
@@ -32,9 +32,9 @@ $ kubectl exec -ti test-pod -- wget kinvolk.io
 On *the first terminal*, you should see:
 
 ```
-NODE            NAMESPACE       POD             CONTAINER       PID     COMM    IP REMOTE                LOCAL                 SENT    RECV
-minikube        default         test-pod        test-pod        134110  wget    4  188.114.96.3:443      172.17.0.2:38190      0       2
-minikube        default         test-pod        test-pod        134110  wget    4  188.114.96.3:80       172.17.0.2:33286      0       1
+NODE             NAMESPACE        POD              CONTAINER        PID       COMM      IP SRC                   DST                   SENT     RECV
+minikube-docker  default          test-pod         test-pod         2176620   wget      4  p/default/test-pod:41 r/1.1.1.1:443         296B     58.12KiB
+minikube-docker  default          test-pod         test-pod         2176618   wget      4  p/default/test-pod:47 r/1.1.1.1:80          70B      381B
```

Comment:
Is this being trimmed without appending the ellipsis? Or did it used 41 or 47 as source ports? (I don't believe so)

Reply:
I think I made a mistake during copy/paste. Fixed it